diff -urN latex2html-99.2alpha12.orig/config/build.pl latex2html-99.2alpha12/config/build.pl
--- latex2html-99.2alpha12.orig/config/build.pl	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/config/build.pl	Wed Sep  8 15:47:39 1999
@@ -1,4 +1,4 @@
-#! /usr/local/bin/perl -w
+#! /usr/bin/perl -w
 
 ###############################################################################
 # $Id$
diff -urN latex2html-99.2alpha12.orig/config/config.pl latex2html-99.2alpha12/config/config.pl
--- latex2html-99.2alpha12.orig/config/config.pl	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/config/config.pl	Wed Sep  8 15:47:39 1999
@@ -1,4 +1,4 @@
-#! /usr/local/bin/perl -w
+#! /usr/bin/perl -w
 
 ###############################################################################
 # $Id$
diff -urN latex2html-99.2alpha12.orig/config/redir.pl latex2html-99.2alpha12/config/redir.pl
--- latex2html-99.2alpha12.orig/config/redir.pl	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/config/redir.pl	Wed Sep  8 15:47:39 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 ###############################################################################
 # $Id$
diff -urN latex2html-99.2alpha12.orig/cweb2html/cweb2html latex2html-99.2alpha12/cweb2html/cweb2html
--- latex2html-99.2alpha12.orig/cweb2html/cweb2html	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/cweb2html/cweb2html	Wed Sep  8 15:47:51 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 ################################################################################
 # Copyright 1998-1999 by Jens Lippmann (lippmann@rbg.informatik.th-darmstadt.de)
diff -urN latex2html-99.2alpha12.orig/cweb2html/makemake.pl latex2html-99.2alpha12/cweb2html/makemake.pl
--- latex2html-99.2alpha12.orig/cweb2html/makemake.pl	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/cweb2html/makemake.pl	Wed Sep  8 15:48:19 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 ################################################################################
 # Copyright 1998-1999 by Jens Lippmann (lippmann@rbg.informatik.tu-darmstadt.de)
--- latex2html-2002-1/configure.orig	2002-04-28 07:15:01.000000000 +0200
+++ latex2html-2002-1/configure	2003-11-23 12:56:07.456729224 +0100
@@ -1159,6 +1159,7 @@
 echo "launching proprietary configuration..."
 
 $PERL $srcdir/config/config.pl \
+  PERL=$PERL \
   HASHBANG=$ac_cv_sys_interpreter \
   OLDCONFIG=$with_oldconfig \
   EXTRAPATH=$with_extrapath \
--- latex2html-2002-1/config/config.pl.orig	2003-11-23 12:38:45.000000000 +0100
+++ latex2html-2002-1/config/config.pl	2003-11-23 12:56:31.550066480 +0100
@@ -601,7 +601,7 @@
 
 # only perl gets an absolute pathname
 my $abs_path_names = 1;
-$newcfg{'PERL'} = &find_prog($^X); # take the perl internal executable name
+$newcfg{'PERL'} = &get_name('PERL'); # take the perl internal executable name
 &checking('perl version');
 &result($]);
 
