diff -urN latex2html-99.2alpha12.orig/config/config.pl latex2html-99.2alpha12/config/config.pl
--- latex2html-99.2alpha12.orig/config/config.pl	Tue Sep  7 21:04:52 1999
+++ latex2html-99.2alpha12/config/config.pl	Wed Sep  8 14:40:32 1999
@@ -505,37 +505,7 @@
 
 &checking('if perl globbing works');
 
-open(TMP,">TEST1.TMP"); close TMP; # create temporary test files
-open(TMP,">TEST2.TMP"); close TMP;
-my @list = (<TEST*.TMP>); # glob test
-unlink qw(TEST1.TMP TEST2.TMP); # remove temporary test files
-
-unless(grep(/^TEST[12]\.TMP$/i,@list) == 2) {
-  &result('no');
-  logit("  Cannot live without globbing\n");
-  # Try to find out what is going wrong
-  # Perl uses full_csh to perform globbing, this may change of course
-  my $csh = eval {require 'Config.pm'; &Config::FETCH('','full_csh')};
-  if($csh) {
-    unless(-f $csh) {
-      logit("Error: $csh does not exist\n");
-    }
-    elsif(!-x $csh) {
-      logit("Error: $csh is not executable\n");
-    }
-    else {
-      logit("Error: unknown globbing problem, check perl installation\n");
-    }
-  }
-  else {
-    logit("Error: perl globbing failed. Couldn't determine csh info.\n");
-  }
-  cleanup(1);
-  exit 1;
-}
-else {
-  &result('yes');
-}
+&result('yes');
 
 # --------------------------------------------------------------------------
 # Hashbang script starts
--- latex2html-2008/config/config.pl~	2013-12-04 08:56:32.000000000 +0100
+++ latex2html-2008/config/config.pl	2013-12-04 08:57:19.314864264 +0100
@@ -1795,8 +1795,6 @@
 my @tmp = ();
 push(@tmp,$cfg{'TMPSPACE'}) if($cfg{'TMPSPACE'});
 push(@tmp,$prefs{'TMPSPACE'}) if($prefs{'TMPSPACE'});
-push(@tmp,$ENV{'TMP'}) if($ENV{'TMP'});
-push(@tmp,$ENV{'TEMP'}) if($ENV{'TEMP'});
 if($newcfg{'plat'} =~ /dos|win32/i) {
   push(@tmp,qw(C:\\TMP C:\\TEMP C:\\WINDOWS\\TEMP C:\\WINNT\\TEMP));
 }
