diff -urN latex2html-99.2alpha12.orig/config/build.pl latex2html-99.2alpha12/config/build.pl
--- latex2html-99.2alpha12.orig/config/build.pl	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/config/build.pl	Wed Sep  8 15:47:39 1999
@@ -1,4 +1,4 @@
-#! /usr/local/bin/perl -w
+#! /usr/bin/perl -w
 
 ###############################################################################
 # $Id$
diff -urN latex2html-99.2alpha12.orig/config/config.pl latex2html-99.2alpha12/config/config.pl
--- latex2html-99.2alpha12.orig/config/config.pl	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/config/config.pl	Wed Sep  8 15:47:39 1999
@@ -1,4 +1,4 @@
-#! /usr/local/bin/perl -w
+#! /usr/bin/perl -w
 
 ###############################################################################
 # $Id$
diff -urN latex2html-99.2alpha12.orig/config/redir.pl latex2html-99.2alpha12/config/redir.pl
--- latex2html-99.2alpha12.orig/config/redir.pl	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/config/redir.pl	Wed Sep  8 15:47:39 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/perl -w
 
 ###############################################################################
 # $Id$
diff -urN latex2html-99.2alpha12.orig/cweb2html/cweb2html latex2html-99.2alpha12/cweb2html/cweb2html
--- latex2html-99.2alpha12.orig/cweb2html/cweb2html	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/cweb2html/cweb2html	Wed Sep  8 15:47:51 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 ################################################################################
 # Copyright 1998-1999 by Jens Lippmann (lippmann@rbg.informatik.th-darmstadt.de)
diff -urN latex2html-99.2alpha12.orig/cweb2html/makemake.pl latex2html-99.2alpha12/cweb2html/makemake.pl
--- latex2html-99.2alpha12.orig/cweb2html/makemake.pl	Wed Sep  8 15:47:09 1999
+++ latex2html-99.2alpha12/cweb2html/makemake.pl	Wed Sep  8 15:48:19 1999
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/perl
 
 ################################################################################
 # Copyright 1998-1999 by Jens Lippmann (lippmann@rbg.informatik.tu-darmstadt.de)
